src/2.1/main.cpp

```diff
@@ -0,0 +1,91 @@
+// BT ve con ma di tuan
+#include <iostream>
+#include <vector>
+#include <iomanip>
+using namespace std;
```

Comment:
Don't use `using namespace std;`, explicitly add the std:: prefix at the front of the objects and functions.